### DISCUSSION TITLE
Feature request for cockpit builders and multi monitor users

### DISCUSSION THREAD

**Discussion by BilSak** [flybywiresim/aircraft#5815] [Ideas]
Hi there, I don't know if this is the right place, and if not please let me know!
I'm a huge fan of FBW and I would love to integrate the model into my home cockpit, built with Skalarki hardware. The issue I'm having is, that I would like to have a clean out of the window cockpit view on my scenery monitor and the e.g. the PFD, ND etc on a separate monitor(s). Now I understand with right alt + left mouse click I can detach and place the relevant displays wherever I want. The issue though is, that it's extremely FPS intense (I guess the reason is, that is has to be rendered twice. E.g. only the MCDU costs about 10-20% in fps) and furthermore the "original" cockpit panels rest in place, thus I would need to move forward in Z axis to "get" out of the cockpit. I think it would be a great feature for all multi-monitor users to be able to start a "separate" app running FBW cockpit panels -> see ProSim Display, or Magenta. I've tried with ProSim A320 but their support does not exist or at least they do not respond to support cases, and Magenta, to what I've heard, is outdated. The best feature, of course, would be a network support environment. So e.g. MSFS running as client to only render scenery, and another computer would run the FBW model,  but I guess the user base is too small to justify the efforts.

**Comment by Gamma36** [comment]
This is the wrong place to ask, as the 3D graphic rendering of the world is done by ASOBO. 

The pop-out displays are ideally 2D which would have minimal load on a GPU. However they are probably 3D renderings at the moment , hence the big hit on overall FPS. We will only know for sure, and get the best performance, when ASOBO support multi monitors including 2D renderings of instruments on separate monitors.

Other solutions are available now, such as Air Manager 2D instruments which have negligible effect on FPS.  

Best to close down this request, as the FBW team will be more concerned with A320 performance, but I hope my comments help you understand the technical issues.
